src/App/component/PullRequestsPage.jsx

```diff
@@ -0,0 +1,170 @@
+import { useEffect, useState } from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import ProjectAvatar from './ProjectAvatar'
+import DrawingBoard from './DrawingBoard'
+import Axios from 'axios'
+import moment from 'moment'
+import { Backdrop, CircularProgress } from '@material-ui/core'
+import { connect } from 'react-redux';
+
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    '& > *': {
+      margin: theme.spacing(1),
+    },
+    minWidth: '30px',
+  },
+  backdrop: {
+    zIndex: theme.zIndex.drawer + 1,
+    color: '#fff',
+  },
+}))
+
+function PullRequestsPage(prop) {
+  const classes = useStyles()
+  const [pullRequestListData, setPullRequestListData] = useState([])
+  const [dataForPullRequestChart, setDataForPullRequestChart] = useState({ labels: [], data: { opened: [], closed: [], merged: [] } })
+
+  const [currentProject, setCurrentProject] = useState({})
+
+  const projectId = localStorage.getItem("projectId")
+  const jwtToken = localStorage.getItem("jwtToken")
+
+  const [open, setOpen] = useState(false);
+  const handleClose = () => {
+    setOpen(false);
+  };
+  const handleToggle = () => {
+    setOpen(!open);
+  };

```

Comment:
Suggest given WHAT is these `open...` used for. (rename)
It may hard to understand who uses this `open` status and state setters.

```suggestion
  const [isWhatOpen, setWhatOpen] = useState(false);
  const handleWhatClose = () => {
    setWhatOpen(false);
  };
  const handleWhatToggle = () => {
    setWhatOpen(!isWhatOpen);
  };

```